board/adi/ev-sc598-som/overlay/init

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
We should make it clear this is only for the bootstrap image. Maybe name the directory `bootstrap_overlay`?